common/test/acceptance/pages/lms/dashboard.py

```diff
@@ -217,11 +217,18 @@ def username_dropdown_link_text(self):
         """
         return self.q(css='.user-dropdown-menu li a').text
 
+    @property
+    def dashboard_tabs_link_text(self):
```

Comment:
Nit: this returns all of the tabs on the dashboard, not just the "dashboard tabs" as defined by "show_dashboard_tabs". I imagine this could be confusing.